cargo-shuttle/src/main.rs

```diff
@@ -106,6 +109,38 @@ impl Shuttle {
         .context("failed to delete deployment")
     }
 
+    async fn init(&self, args: InitArgs) -> Result<()> {
+        // Call `cargo init --lib` to generate files
+        let path = args.path.to_str().unwrap();
+        process::Command::new("cargo")
+            .args(["init", "--lib", path])
+            .spawn()?
+            .wait()?;
```

Comment:
I know the cargo API documentation is a bit lacking, but maybe we can interface with it directly?
https://docs.rs/cargo/latest/cargo/ops/fn.init.html

Reply:
Almost forgot. Here is an example of us interacting with `cargo` directly
https://github.com/shuttle-hq/shuttle/blob/main/api/src/build.rs#L213